### ISSUE TITLE
[Button] Problem with button's color in rtl build

### ISSUE THREAD

**Issue by mjnaderi** [Semantic-Org/Semantic-UI#3750]
Hi,

Simple buttons have no color in rtl build.

```html
<body dir="rtl">
<div class="ui button">This has no color</div>
<div class="ui blue button">Blue Button</div>
</body>
```

![screenshot from 2016-02-24 12-13-57](https://cloud.githubusercontent.com/assets/2115303/13279929/332a60da-daf0-11e5-9926-88070f65ad67.png)

JSFiddle: https://jsfiddle.net/r0fmcmL7/

**Comment by MohammadYounes** [comment]
This issue was fixed in [RTLCSS 2.0.2](https://github.com/MohammadYounes/rtlcss/blob/master/CHANGELOG.md#202---05-mar-2016), Update your local packages and rebuild.

Thanks!

**Comment by mjnaderi** [comment]
It is fixed now. Thanks!
